### DISCUSSION TITLE
I am seeing an error when run i ! nvidia-smi in code block in google co lab ?

### DISCUSSION THREAD

**Discussion by tedzz12** [mrdbourke/pytorch-deep-learning#321] [General]
I am seeing this error:- NVIDIA-SMI has failed because it couldn't communicate with the NVIDIA driver. Make sure that the latest NVIDIA driver is installed and running.when i run  ! nvidia-smi in code block in google co lab  how t solve this error?

**Comment by karthikbhandary2** [comment]
Did you set the hardware accelerator as "GPU"?

**Reply by tedzz12** [reply]
yes,i put GPU in hardware accelerator but keep seeing the error.

**Reply by karthikbhandary2** [reply]
Take a look here: https://forums.developer.nvidia.com/t/nvidia-smi-has-failed-because-it-couldnt-communicate-with-the-nvidia-driver-make-sure-that-the-latest-nvidia-driver-is-installed-and-running/197141/2

Let me know if there is anything

**Reply by tedzz12** [reply]
thank you

**Reply by tedzz12** [reply]
its working now

**Reply by karthikbhandary2** [reply]
great!
